### ISSUE TITLE
Form styles.

### ISSUE THREAD

**Issue by WMontagnon** [Qarzak/nuCSS#8]
Create styles for forms and it's children :
- [x] Form
- [ ] Inputs
- [ ] Text Areas
- [ ] Select
- [ ] Checkboxes
- [ ] Radio buttons
- [ ] Fieldsets

**Comment by pv2k** [comment]
I would love to work on this!!!! :)

**Comment by WMontagnon** [comment]
Go ahead, and don't hesitate to ask any questions.

**Comment by code4cake** [comment]
@pv2k are you working on this? Do you need any help?

**Comment by pv2k** [comment]
Yea. I have started working on it. If there are any doubts,I will ask them here :smiley:

**Comment by pv2k** [comment]
#31 PR Form styles added.

**Comment by WMontagnon** [comment]
It's alright for forms, let's now make it for its children. 

@pv2k Are you still working on this, or can I drop the `claimed` label ?

**Comment by pv2k** [comment]
@Qarzak, I thought fieldsets,inputs....  are the forms children. Is that right???

**Comment by WMontagnon** [comment]
Yeah that's right, anything that is listed in the first comment of this issue, and maybe more.

**Comment by pv2k** [comment]
Okay!!! I will work on them. :smiley:
